packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1291,6 +1291,16 @@ export const RewriteWithBodyUsed = {
 		'Astro.rewrite() cannot be used if the request body has already been read. If you need to read the body, first clone the request.',
 } satisfies ErrorData;
 
+/**
+ * @docs
+ * @description
+ * An unknown error occured while reading or writing files to disk.
```

Comment:
Some examples of why this can happen would be great, just to give a hint to the user. You can also mention that more information is available in the `cause`

Reply:
Have examples?

Reply:
There's a few common errors that can happen, like missing permissions, or a file not existing when it's supposed to.

Notably permissions are sometimes an head scratcher for Windows users (ex: they put their Astro project in a weird folder)